### ISSUE TITLE
Unable to install... Unknown MySQL server host 'SQL_SERVER' (2)

### ISSUE THREAD

**Issue by Batx** [yi12345/TravianT4#76]
Hey, always getting this Error while installing... Hope someone can help :)

ERROR: "Unknown MySQL server host 'SQL_SERVER' (2)"
MySQL password and login always was correct... I tripple checked it...
Regards and thanks!

**Comment by cosme12** [comment]
Maybe your sql url isnt right. Did you write "localhost" or the url may be wrong.

**Comment by Batx** [comment]
did localhost
does not work...

**Comment by Batx** [comment]
:(

**Comment by cosme12** [comment]
Ask your server host what is your mySQL url.

**Comment by Batx** [comment]
its my server i have root rights

**Comment by Batx** [comment]
so localhost should work!

**Comment by cosme12** [comment]
Try 127.0.0.1 insteed. Maybe it works

**Comment by Batx** [comment]
now it ran untill /install/include/multihunter.php and then the error came agian... Unknown MySQL server host 'SQL_SERVER' (2)
The databese is even getting filled up with everything... so the connection works...

**Comment by cosme12** [comment]
Yeah, i think this version had troubles with multihunter. Go to the database and create the user (it may be already created) manually. That will allow you to finish the installation. 

Keep in mind that this T4 version is full of bugs. I only know a T3 version fully working but not a T4

**Comment by Batx** [comment]
yeah, the thing is no one likes t3... ;(

**Comment by Batx** [comment]
The install multihunter is on install/index.php?s=4 after trying it again, i got the error again. then i went to the url install/index.php?s=5 (Next step oasis)
/install/include/oasis.php
but there i again got this error... idk what to do...

Regards 👍 
Batx

**Comment by cosme12** [comment]
It displays the error but still creates the thing if im not wrong. Keep typing in the url s6 till its done

**Comment by cosme12** [comment]
Also, this is the T4 version. I think the 4.2 changed a lot and improved a lot of features. Someone should work on a new version :P

**Comment by Batx** [comment]
Yes :D Someone should do that :P Are you good at that? Im having someone, but he does not have so much time... And my skills are limited :D

**Comment by cosme12** [comment]
Depends on your definition of "good" :D Let me find a 4.2 version to start with so I can create a repository and you can help me. You should be a professional coder by the end of this :P

**Comment by cosme12** [comment]
Well, this is the only 4.2 script I found. Lets do this:
1) download it here: https://www.mediafire.com/?d234sgc2lbjha8j
2) create a new repository here in github (so me and your friend can see it and you can learn how this works)
3) once its done, we can all install it and start modifying it, so if any of us makes a change, everybody can see it.
4) main problem, its no fully in english

If you have any better idea, just let me know.

Edit: I found a better script.
Edit2: this is still a v4 script :/  We will have to keep searching for a 4.2 version :(

**Comment by Batx** [comment]
Oh... So that is no 4.2 script... Hmm do you have skype or steam? Maybe its better to have a chat :D

Regards

**Comment by Batx** [comment]
So now i have installed it, http://www.letsplay.tv/ but login and register is not working... And now errors are shown up...

**Comment by cosme12** [comment]
I have a little fixed version of T4 with installation fixed. https://github.com/cosme12/TravianT4 its the same as this version but with some bugs fixed, mostly installation. But still its pretty broken. If we want to make it look like T4.4 it will take a lot of work.

**Comment by cosme12** [comment]
Here is my skyp, add it fast so i dont get bombarded with spam :P diegolan894

**Comment by Batx** [comment]
Added u :)
